jupyterhub/templates/scheduling/user-scheduler/configmap.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.scheduling.userScheduler.enabled -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: user-scheduler
+  labels:
+    {{- include "jupyterhub.labels" . | nindent 4 }}
+data:
+  policy.cfg: |
+    {
+      "kind": "Policy",
+      "apiVersion": "v1",
+      "predicates": [
+        { "name": "PodFitsResources" },
+        { "name": "HostName" },
+        { "name": "PodFitsHostPorts" },
+        { "name": "MatchNodeSelector" },
+        { "name": "NoDiskConflict" },
+        { "name": "PodToleratesNodeTaints" },
+        { "name": "MaxEBSVolumeCount" },
+        { "name": "MaxGCEPDVolumeCount" },
+        { "name": "MaxAzureDiskVolumeCount" },
+        { "name": "CheckVolumeBinding" },
+        { "name": "NoVolumeZoneConflict" },
+        { "name": "MatchInterPodAffinity" }
+      ],
+      "priorities": [
+        { "name": "NodePreferAvoidPodsPriority",  "weight": 10000 },
+        { "name": "NodeAffinityPriority",         "weight": 1000 },
+        { "name": "InterPodAffinityPriority",     "weight": 100 },
+        { "name": "MostRequestedPriority",        "weight": 50 }
+      ],
+      "hardPodAffinitySymmetricWeight" : 100,
+      "alwaysCheckAllPredicates" : true
+    }
+{{- end }}
```

Comment:
These are the actual configuration that makes the user-scheduler special and different from the default scheduler.